### DISCUSSION TITLE
"Page Not Found" error for revealjs slides published on Quarto Pub

### DISCUSSION THREAD

**Discussion by kyeager4** [quarto-dev/quarto-cli#7445] [Q&A] [answered]
### Description

I am attempting to publish revealjs slides to my Quarto Pub account. When I run `quarto publish quarto-pub` in the terminal inside RStudio, the process appears to go through successfully, and when I look at my Quarto Pub public landing page (https://kyeager330.quarto.pub/), I see the title slide appearing as intended. However, when I click the link to view them, I see the "Page Not Found" error message (https://kyeager330.quarto.pub/positconf2023-brownbag-fpdc/). The issue does not appear to be limited to my revealjs slides: I tried generating a new instance of the the default Quarto Website project in RStudio and uploading it to Quarto Pub, and I'm am getting the same "Page Not Found" issue there (https://kyeager330.quarto.pub/test2/).

The issue appears to be similar (identical?) to #6419 (as well as #4386, #3365, and #3362): I see the same "Page Not Found" message screenshotted in those threads). In the case of #6419 there seems to have been a configuration change on the back-end that was needed to resolve the issue. Does anyone know if that issue (or a similar issue) may be happening again?

I've deleted the project from Quarto Pub and attempted to re-upload it several times (removing the old _publish.yml file before I do) to no avail. I've also tried renaming the *.qmd file to index.qmd (no luck) and using a fresh RStudio project with a different name (no luck).

**Comment by lawtj** [comment]
Perhaps it's more widespread - I have the same issue even when publishing 'hello.qmd' from the getting started docs even with fresh accounts; all of the docs I've tried show 'page not found'.

**Comment by mcanouil** [comment]
We are aware of the issue and investigating.
Note that the issue is not user side.

**Comment by softwarenerd** [comment, marked as answer]
The live site issue with Quarto Pub has been resolved. Please just re-publish one of the sites you recently published, and everything should work thereafter.
